docs/metrics.md

```diff
@@ -42,6 +41,29 @@ When we aggregate Lead time for changes we use the maximum of a period of time.
 | High                  | Between 1 day and 1 week     |
 | Elite                 | Less than 1 day              |
 
+### Sub-metrics
+
+There are several metrics that directly influence the lead time for changes,
+and can help you track in more detail what needs to be improved in your workflow.
+All the sub-metrics are aggregated by an interval (day, week or month) and have values for
+the average, median, and percentiles 5%, 20%, 25%, 75%, 80%, 95%.
+
+These metrics are extracted from pull requests,
+only include pull requests that have been merged (ignores closed without merge and currently open) and
+draft pull requests are treated as normal pull requests.
+
+#### Time to Open
+
+An approximation of the time a change takes to be developed.
+This metric assumes you are not developing after the PR is open.
```

Comment:
assumes but takes those PRs into account

Reply:
Since we stopped calling time in development this probably makes no sense

Reply:
```suggestion
```

Reply:
other tools solve this problem by assuming that commits after the PR is opened is rework, and have different metrics for that.
```suggestion

```